airflow/cli/commands/connection_command.py

```diff
@@ -112,17 +112,13 @@ def _format_connections(conns: List[Connection], fmt: str) -> str:
 
 
 def _is_stdout(fileio: io.TextIOWrapper) -> bool:
-    if fileio.name == '<stdout>':
-        return True
-    return False
+    return fileio.name == '<stdout>'
 
 
 def _valid_uri(uri: str) -> bool:
     """Check if a URI is valid, by checking if both scheme and netloc are available"""
     uri_parts = urlparse(uri)
-    if uri_parts.scheme == '' or uri_parts.netloc == '':
-        return False
-    return True
+    return uri_parts.scheme != '' or uri_parts.netloc != ''
```

Comment:
This should be changed into `uri_parts.scheme != '' and uri_parts.netloc != ''`, instinct from a math-major guy ;-)

Reply:
Right, De Morgans laws - and I did a thesis from fuzzy logic 🤦

Reply:
Ha, only now I know you also majored in math😄 My focus was random process and application in bio-stats.

But anyway, not rare for math people like us to miss OR/AND time to time😄